src/Webhooks.ts

```diff
@@ -1,10 +1,12 @@
 'use strict';
 
-const utils = require('./utils');
-const {StripeError, StripeSignatureVerificationError} = require('./Error');
+import utils = require('./utils');
+import _Error = require('./Error');
+const {StripeError, StripeSignatureVerificationError} = _Error;
 
 const Webhook = {
   DEFAULT_TOLERANCE: 300, // 5 minutes
+  signature: null,
```

Comment:
To fix property does not exist error at https://github.com/stripe/stripe-node/blob/84d3ec43b0dd582fee811f2330f2001095e90475/src/Webhooks.ts#L277